bedrock/newsletter/templates/newsletter/includes/form-protocol.html

```diff
@@ -0,0 +1,113 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% set_lang_files "mozorg/newsletters" %}
+
+{% from "newsletter/includes/macros.html" import email_form_thankyou with context %}
+
+{% if not success %}
+  <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post">
+    {{ form.newsletters|safe }}
+    <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
+
+    {% if include_title %}
+    <header class="mzp-c-newsletter-header">
+      <h3 class="mzp-c-newsletter-title">{{ title|d(_('Get Firefox news'), true) }}</h3>
+      {% if subtitle %}
+      <p class="mzp-c-newsletter-tagline">{{ subtitle }}</p>
+        {% endif %}
+    </header>
+    {% endif %}
+
+    <fieldset class="mzp-c-newsletter-content">
+      {% if form.errors %}
+      <div class="mzp-c-form-errors" id="newsletter-errors">
+        {{ form.non_field_errors()|safe }}
+        <ul class="mzp-u-list-styled">
+          {% if form.email.errors %}
+            <li>{{ _('Please enter a valid email address') }}</li>
+          {% endif %}
+
+          {% if form.lang.errors %}
+            <li>{{ form.lang.errors }}</li>
+          {% endif %}
+
+          {% if form.privacy.errors %}
+            <li>{{ _('You must agree to the privacy notice') }}</li>
+          {% endif %}
+        </ul>
+      </div>
+      {% endif %}
+
+      <p>
+        <label for="id_email">{{ _('Your email address') }}</label>
```

Comment:
@peiying2 This PR doesn't have strict l10n requirements as it's for English locales only. But there are a few new strings being added to our newsletter form here. It would be good to start translating these if possible (newsletter strings only), so by the time we come to use this on a translated page we're in good shape? Let me know if you need anything here.

Reply:
This is being worked on. Thanks for the heads-up.